client/src/index.tsx

```diff
@@ -6,11 +6,15 @@ import App from "./app"
 import {IFabricExports} from "./body/fabric-exports"
 import "./index.css"
 import registerServiceWorker from "./service-worker"
-import {LocalStorage} from "./storage/local-storage"
+import {BrowserStorage} from "./storage/browser-storage"
+import {RemoteStorage} from "./storage/remote-storage"
+import {IStorage} from "./storage/storage"
 
+const REMOTE_STORAGE = false
```

Comment:
There's a way to read this from Node environment variables using Webpack. https://stackoverflow.com/questions/49616291/process-env-on-client-side-reactjs